src/web/mjs/connectors/Nosajj.mjs

```diff
@@ -0,0 +1,52 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Nosajj extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'Nosajj';
+        super.label = 'Nosajj (shonanjunaigumi)';
+        this.tags = ['manga', 'english', 'scanlation'];
+        this.url = 'http://shonanjunaigumi.weebly.com';
+    }
+
```

Comment:
Website lacks an icon. Since website have default weebly icon you may copy default hakuneko icon (\\img\connectors\default) and rename it to match the website id.